### PR TITLE
set custom name for hwloc lib

### PR DESCRIPTION
**Pull request by bratpiorka**
### Description

If hwloc is got from vcpkg it is named "hwloc" while mingw version names it "libhwloc".

### Checklist

- [ ] Code compiles without errors locally
- [ ] All tests pass locally
- [ ] CI workflows execute properly

### CODE REVIEW

File: .github/workflows/basic.yml [hunk 1]
```diff
@@ -348,6 +348,49 @@ jobs:
...
+    - name: Get TBB from github
+      run: |
+        curl https://github.com/oneapi-src/oneTBB/releases/download/v2021.12.0/oneapi-tbb-2021.12.0-win.zip -o ${{github.workspace}}\oneapi-tbb-2021.12.0-win.zip
+        Expand-Archive ${{github.workspace}}\oneapi-tbb-2021.12.0-win.zip -DestinationPath ${{github.workspace}}
+  
+    - name: Configure build
+      run: >
+        cmake
+        -B ${{env.BUILD_DIR}}
+        -DCMAKE_INSTALL_PREFIX="${{env.INSTL_DIR}};${{github.workspace}}\hwloc-win64-build-2.10.0;${{github.workspace}}\oneapi-tbb-2021.12.0"
```
Comment (lukaszstolarczuk) [inline .github/workflows/basic.yml:386]: install prefix should be set to `-DCMAKE_INSTALL_PREFIX="${{env.INSTL_DIR}}"`
what you want to set up here is `-DCMAKE_PREFIX_PATH="tbb_dir;hwloc_dir"

File: .github/workflows/basic.yml [hunk 2]
```diff
@@ -348,6 +348,49 @@ jobs:
...
+        -DUMF_BUILD_SHARED_LIBRARY=ON
+        -DUMF_BUILD_EXAMPLES=ON
+        -DUMF_FORMAT_CODE_STYLE=ON
```
Comment (lukaszstolarczuk) [inline .github/workflows/basic.yml:389]: code_style=off

File: .github/workflows/basic.yml [hunk 3]
```diff
@@ -348,6 +348,49 @@ jobs:
...
+
+    runs-on: 'windows-2022'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: 0
+
+    - name: Get hwloc from offitial repo (mingw version)
```
Comment (lukaszstolarczuk) [inline .github/workflows/basic.yml:372]: nit: official (mispell)

File: .github/workflows/basic.yml (hunk 1 above)
Comment (bratpiorka) [reply .github/workflows/basic.yml]: done

File: .github/workflows/basic.yml (hunk 2 above)
Comment (bratpiorka) [reply .github/workflows/basic.yml]: done

File: .github/workflows/basic.yml (hunk 3 above)
Comment (bratpiorka) [reply .github/workflows/basic.yml]: done

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**